src/main/webapp/view/video/video.js

```diff
@@ -38,15 +46,16 @@ export default class Video {
       width: '640',
       videoId: window.LECTURE.videoId,
       events: {
-        onReady: this.onPlayerReady,
+        onReady: window.onPlayerReady,
```

Comment:
which is this window now? the method is declared in this class.

Reply:
The window is needed for the API to be able to call this

Reply:
talked offline and Jay explained why this is needed.